### ISSUE TITLE
automatically create kernels for tutorial using jupyterlab-requirements-cli in a step in aicoe-pipeline

### ISSUE THREAD

**Issue by pacospace** [AICoE/meteor#35]
This could be done after image is spawned (checking dependencies from the notebooks) and install those kernels so they are ready for the users. Even though it will slow down a bit the spawning process

**Comment by pacospace** [comment]
Related-To: https://github.com/thoth-station/jupyterlab-requirements/issues/324

**Comment by pacospace** [comment]
cc @MichaelClifford

**Comment by tumido** [comment]
I'm confused @pacospace if this should be done after the image is spawned, it can't be part of the build pipeline. Can you please clarify when you expect this step to happen?

If it's part of the build phase, can this be included as a custom s2i assembly script? It seems fairly specific for this given use case/repository, isn't it?

**Comment by pacospace** [comment]
> I'm confused @pacospace if this should be done after the image is spawned, it can't be part of the build pipeline. Can you please clarify when you expect this step to happen?
> 
> If it's part of the build phase, can this be included as a custom s2i assembly script? It seems fairly specific for this given use case/repository, isn't it?

Yes build process sounds good, s2i assembly script seems to be the right place.

In this way, if notebooks are missing dependencies, the build process will fail also as those are not reproducible.

**Comment by pacospace** [comment]
`horus`: jupyterlab-requirements CLI -> https://github.com/thoth-station/jupyterlab-requirements#horus-jupyterlab-requirements-cli
